src/frontend/planner_test/tests/testdata/nexmark.yaml

```diff
@@ -473,9 +473,7 @@
         WHERE A.id = B.auction and B.date_time between A.date_time and A.expires
         GROUP BY A.id, A.seller
     ) AS Q;
-  binder_error: |-
-    Feature is not yet implemented: aggregate function as over window function: avg
-    Tracking issue: https://github.com/risingwavelabs/risingwave/issues/4978
+  planner_error: 'Invalid input syntax: column must appear in the GROUP BY clause or be used in an aggregate function'
```

Comment:
The error seems unclear :eyes:

Reply:
This seems to be an unsupported situation not related to window function.